### ISSUE TITLE
Responsive Table pattern loses semantics at narrow size

### ISSUE THREAD

**Issue by aardrian** [cehfisher/a11y-style-guide#257]
## Issue
For the responsive table on this page:
https://a11y-style-guide.com/style-guide/section-structure.html

When the narrow viewport styles kick in, the semantics are lost. This is a known issue across browsers ([back to 2011](https://www.456bereastreet.com/archive/201111/screen_readers_and_css/) and [again in 2018](https://developer.paciellogroup.com/blog/2018/03/short-note-on-what-css-display-properties-do-to-table-semantics/)).

This means a screen reader user will get no structure, and will be unable to navigate the content as a table.

ARIA table roles can fix this until browsers fix their bugs. I have a [function authors can use](http://adrianroselli.com/2018/05/functions-to-add-aria-to-tables-and-lists.html) in the meantime. Cannot find bugs right now.

The accessibility inspector in Chrome/Windows showing the example table in a large viewport. The table structure is visible.
![2019-09-05 (1)](https://user-images.githubusercontent.com/1376607/64342844-5dbbd100-cfb9-11e9-9de0-0143409bcefb.png)

The accessibility inspector in Chrome/Windows showing the example table in a narrow viewport. The table structure is now absent.
![2019-09-05 (2)](https://user-images.githubusercontent.com/1376607/64342872-6e6c4700-cfb9-11e9-9a7f-5fe142087f7b.png)

**Comment by cehfisher** [comment]
Yes, responsive tables have some issues to work out for sure. I'll look into implementing the pattern updates you suggest and will send it back to you for a quick review soon. Thanks!
